### PR TITLE
Centos 8 dockerfile

### PR DESCRIPTION
**Pull request by TedThemistokleous**
Support for Centos 8

related to things with adding more support. Similar to #2029

### CODE REVIEW

File: tools/docker/centos_8.dockerfile [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+RUN sh -c 'echo yum config-manager --set-enabled PowerTools'
+
+# Install dependencies
+RUN dnf update  -y && dnf makecache --refresh
+RUN dnf groupinstall -y "Development Tools" --setopt=group_package_types=mandatory,default,optional
+
+RUN dnf --enablerepo=powertools install -y --nobest \
+    yum-utils \
+    #build-essential \
+    #clang-format-10 \
```
Comment (causten) [inline tools/docker/centos_8.dockerfile:31]: There are a number of these lines commented out.  Just remove them

**Review by causten — CHANGES_REQUESTED** [review]

File: tools/docker/centos_8.dockerfile (hunk 1 above)
Comment (TedThemistokleous) [reply tools/docker/centos_8.dockerfile:31]: will do

Comment (TedThemistokleous) [reply tools/docker/centos_8.dockerfile]: Done

**Comment by causten** [comment]
ROCm only support CentOS 7.9, not 8 https://rocm.docs.amd.com/en/latest/release/gpu_os_support.html

**Comment by TedThemistokleous** [comment]
Closing this out as I'll need to figure out the pieces for Centos-7.9. Centos8 not needed

**Pull request closed without merging**